__tests__/app.test.js

```diff
@@ -65,4 +65,34 @@ describe('/api/articles/:article_id', () => {
             .expect(400)
         expect(msg).toBe("Bad request")
     })
-})
\ No newline at end of file
+})
+
+describe('/api/articles', () => {
+    test('200: Should return an array of articles', async () => {
+        const { body: { articles } } = await request(app)
+            .get("/api/articles")
+            .expect(200)
+        expect(articles).toHaveLength(13)
+        articles.forEach(article => {
+            expect(article).toHaveProperty("author", expect.any(String))
+            expect(article).toHaveProperty("title", expect.any(String))
+            expect(article).toHaveProperty("article_id", expect.any(Number))
+            expect(article).toHaveProperty("votes", expect.any(Number))
+            expect(article).toHaveProperty("topic", expect.any(String))
+            expect(article).toHaveProperty("created_at", expect.any(String))
+            expect(article).toHaveProperty("article_img_url", expect.any(String))
+            expect(article).toHaveProperty("comment_count", expect.any(String))
```

Comment:
You have comment count coming out as a string. There is a way to make it a number. Not mega important for this sprint but something you can look into 🙂